.core_files.yaml

```diff
@@ -0,0 +1,120 @@
+# Defines a list of files that are part of main core of Home Assistant.
+# Changes to these files/filters define how our CI test suite is ran.
+core: &core
+  - homeassistant/*.py
+  - homeassistant/auth/**
+  - homeassistant/helpers/*
+  - homeassistant/package_constraints.txt
+  - homeassistant/util/*
+  - pyproject.yaml
+  - requirements.txt
+  - setup.cfg
+
+# Our base platforms, that are used by other integrations
+base_platforms: &base_platforms
```

Comment:
In the future maybe we can use the dependency resolution code to generate these

Reply:
Base platforms are not bound by dependecies.

Reply:
What about `"quality_scale": "internal"` ?

Reply:
I probably needed to give a bit more context here:

This list looks quite similar to:
https://github.com/home-assistant/core/blob/dev/script/hassfest/dependencies.py#L93


Related:
https://github.com/home-assistant/core/blob/dev/script/hassfest/dependencies.py#L7

```
from homeassistant.requirements import DISCOVERY_INTEGRATIONS
from homeassistant.setup import BASE_PLATFORMS
```

Reply:
> What about "quality_scale": "internal" ?

What about it? I really don't want to trigger a full suite if "season" or "uptime" changes...

> I probably needed to give a bit more context here:

yes, the base platforms are defined in our codebase as well (and are actually incomplete as we have discovered today). I guess having a single source might make sense, right now, I'm kinda happy it is contained in the CI itself actually... (regardless of code).

Reply:
If this all turns out to work as expected, I'll adjust the CI to get the base & discovery integrations from Python, opened up a PR to adjust hassfest to have a matching list as well.